src/elements/kw-view-projects/kw-view-projects.html

```diff
@@ -251,6 +255,36 @@
             _userChanged (user) {
                 this._upgradeProjects(this.categories, this.user);
             },
+            _isPi () {
+                let userAgent = window.navigator.userAgent;
+                return userAgent.indexOf('armv6l') !== -1 || userAgent.indexOf('armv7l') !== -1;
+            },
+            _checkAvailableDependencies () {
+                let availableDependencies = [];
+                if (!this._isPi()) {
+                    availableDependencies = [
+                        'camera',
+                        'microphone',
+                        'text-to-speech'
+                    ];
+                }
+                this.set('availableDependencies', availableDependencies);
+            },
+            _checkCategoryDependencies (category) {
+                let availableProjects = category.projects.filter((project) => {
+                    let allDependenciesAvailable = true;
+                    if (project.dependencies) {
+                        project.dependencies.forEach((dependency) => {
+                            if (this.availableDependencies.indexOf(dependency) === -1) {
```

Comment:
Fast reference to MDN if you need it: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every